### PR TITLE
New Eslint

### PR DESCRIPTION
**Pull request by tjs-intel**
Tested the substantial changes ~I need to poke around in a few places and make sure I didn't break anything:~
 * [x] experiment history page
 * [x] experiment parameter editor for conditionals
 * [x] change password page

### CODE REVIEW

**Comment by tjs-intel** [comment]
I definitely broke the password change page

File: src/python/zigopt/authentication/login.py [hunk 1]
```diff
@@ -13,13 +13,11 @@
 from zigopt.user.model import User, do_password_hash_work_factor_update, password_matches
 
 
-@deal.has()
```
Comment (dwanderson-intel) [inline src/python/zigopt/authentication/login.py:16]: unrelated changes? fine to remove if needed, but seems separate

**Review by dwanderson-intel — APPROVED** [review]
spot checked, seemed to mostly be `&&` --> `? ... : null`, which seems more explicit (good!)

File: src/python/zigopt/authentication/login.py (hunk 1 above)
Comment (tjs-intel) [reply src/python/zigopt/authentication/login.py:16]: Yeah the API was breaking locally because of a contract error

**Pull request merged**